packages/ra-ui-materialui/src/list/SimpleList/SimpleList.tsx

```diff
@@ -272,47 +154,112 @@ export interface SimpleListProps<RecordType extends RaRecord = any>
     total?: number;
 }
 
-const LinkOrNot = (
-    props: LinkOrNotProps & Omit<ListItemProps, 'button' | 'component' | 'id'>
+const SimpleListItemContent = <RecordType extends RaRecord = any>(
```

Comment:
I'd like to extract this to its own file but it uses `SimpleListClasses`

Reply:
To avoid recursive imports. Another option would be to extract the SimpleList styled root in its own file

Reply:
Got it. Let's keep it that way for now.